utils/create_dummy_models.py

```diff
@@ -378,9 +385,18 @@ def get_tiny_config(config_class, **model_tester_kwargs):
         module_name = model_type_to_module_name(model_type)
         if not modeling_name.startswith(module_name):
             raise ValueError(f"{modeling_name} doesn't start with {module_name}!")
-        module = importlib.import_module(f".models.{module_name}.test_modeling_{modeling_name}", package="tests")
-        camel_case_model_name = config_class.__name__.split("Config")[0]
-        model_tester_class = getattr(module, f"{camel_case_model_name}ModelTester", None)
+        test_file = os.path.join("tests", "models", module_name, f"test_modeling_{modeling_name}.py")
+        models_to_model_testers = get_model_to_tester_mapping(test_file)
+        # Find the model tester class
+        model_tester_class = None
+        tester_classes = []
+        if model_class is not None:
+            tester_classes = get_tester_classes_for_model(test_file, model_class)
+        else:
+            for _tester_classes in models_to_model_testers.values():
+                tester_classes.extend(_tester_classes)
+        if len(tester_classes) > 0:
+            model_tester_class = sorted(tester_classes, key=lambda x: x.__name__)[0]
```

Comment:
The new approach re-use some methods in another script (recently added) `utils/get_test_info.py` to get all model tester classes in a test file. And try to get one for the target configuration class here.

- if `model_class` is specified, narrow down the candidates
- use the class with shortest name

Reply:
To be transparent: The new argument `model_class` is not really used in the callers in this script. It requires much more work to use this new argument, and we don't earn much by this change.

The reason that I add this is just it is the **theoretically** correct way - to prevent `GPTSanJapaneseForConditionalGeneration` to use the config given by `GPTSanJapaneseTester`.

**(I agree it's probably nothing at all)**